docs/New Game.md

```diff
@@ -152,3 +152,28 @@ misconfiguration in Randovania.
 Before a game can work in Multiworld, it must first work properly as a solo game. It is highly recommended to finish the basic integration first.
 
 **TODO**
+
+# Expectations of a Development Game
+
+After a game has been merged and is available in development builds, the responsibility of maintaining it is now shared between you and the Randovania team:
+- Randovania: We make sure your game is included in all Randovania changes.
+- You: Work towards the game reaching stable.
+
+We're happy to support new games by doing our part, but since that has a cost we don't want to be doing it forever there's some expectations regarding time frames:
```

Comment:
This sentence right now sounds weird
```suggestion
We're happy to support new games by doing our part, but since that has a cost, that we don't want to be doing forever, there are some expectations regarding time frames:
```